install/03_rbac.yaml

```diff
@@ -10,10 +10,7 @@ rules:
   - validatingwebhookconfigurations
   - mutatingwebhookconfigurations
   verbs:
-  - get
-  - patch
-  resourceNames:
-  - autoscaling.openshift.io
+  - "*"
```

Comment:
Do we need mutating webhook permissions?

Reply:
Not yet. We can remove that for now. Done.